packages/components/src/Table/Table.css

```diff
@@ -2,13 +2,28 @@
   --border-color: var(--color-grey--lighter);
   --header-border-color: var(--color-blue);
   --horizontal-inset: var(--space-base);
+  position: relative;
   width: 100%;
-
-  box-shadow: 0 0 0 var(--border-base) var(--border-color);
   border-collapse: collapse;
+}
+
+.table::after {
+  content: "";
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: 1;
+  width: calc(100% - 1px);
+  height: 100%;
+  border: solid 1px var(--border-color);
```

Comment:
Can we do `solid var(--border-base) var(--border-color)`?

Reply:
Also if the 1px on line 16 is also based on desired border width, this would apply there as well

Reply:
changed it to css custom properties